Resources/Prototypes/Entities/Structures/Furniture/potted_plants.yml

```diff
@@ -235,6 +235,14 @@
   - type: Sprite
     state: plant-25
 
+- type: entity
+  id: PottedPlant30
+  parent: PottedPlantBase
+  description: Is it just me, or is it blinking?
+  components:
+  - type: Sprite
+    state: plant-30
+
```

Comment:
Any reason this isn't in order with the rest?
Since it also inherits the same parent.